firestore/query.go

```diff
@@ -683,6 +666,14 @@ func (it *DocumentIterator) GetAll() ([]*DocumentSnapshot, error) {
 		}
 		docs = append(docs, doc)
 	}
+	if limitedToLast {
+		// Flip docs order before return.
+		for i, j := 0, len(docs)-1; i < j; {
+			docs[i], docs[j] = docs[j], docs[i]
+			i++
+			j--
+		}
+	}
 	return docs, nil
 }
 
```

Comment:
Just to confirm, are methods in queryDocumentIterator (below) affected by adding limit to last? For example, should queryDocumentIterator.next() also return an error when called on a limitToLast query?

Reply:
I've reworked the test to check that `DocumentIterator.Next()` call will fail with `LimitToLast` set. This method actually delegates to `queryDocumentIterator.next()`, so it'll fail even before `queryDocumentIterator.next()`.

Reply:
Cool, thanks for confirming.